### PR TITLE
ignore whitespace in Dangerbot and change LOC maximums by 20%

### PR DESCRIPTION
**Pull request by thilton-oddball**
## Description of change
Altered Dangerfile to ignore whitespace when determining lines of code (LOC) since this behavior is not available with Danger. Also reduced the LOC maximums by 20% since whitespace will no longer be a factor.

## Original issue(s)
department-of-veterans-affairs/va.gov-team/15233

### CODE REVIEW

File: Dangerfile
Comment (erluti) [inline Dangerfile:4]: I think the RECOMMENDED_MAXIMUM of 200 makes sense, but i think without whitespace counting we should push the ABSOLUTE_MAXIMUM even lower.  I don't think I've seen many PRs approach the 1k that didn't need to be broken up, and without whitespace changes I bet it could be a lower maximum.  But I think I guess I also think we'll get data faster if we set it too low and have to increase it.
I would put it at 500, but I'm going to solicit some more feedback.

File: Dangerfile
Comment (erluti) [inline Dangerfile:15]: i think the message output by dangerbot below should also mention it's not counting whitespace changes.

File: Dangerfile
Comment (kfrz) [reply Dangerfile:4]: I think 500 is good middle ground but without taking a look at historical danger-warnings it's sort of guess-n-check anyhow.

**Review by kfrz — APPROVED** [review]
LGTM. Will approve so you can merge but feel free to wait for additional feedback from the team.

**Review by erluti — APPROVED** [review]
LGTM
gotta make sure we keep up with PRs after this is merged incase we end up blocking PRs regularly

**Pull request merged**
